openapi_python_client/templates/endpoint_macros.py.jinja

```diff
@@ -11,6 +11,20 @@ if {{ parameter.python_name }} is not UNSET:
 {% endif %}
 {% endmacro %}
 
+{% macro cookie_params(endpoint) %}
+{% if endpoint.cookie_parameters %}
+    {% for parameter in endpoint.cookie_parameters %}
+        {% if parameter.required %}
+cookies["{{ parameter.name}}"] = {{ parameter.python_name }}
+        {% else %}
+if {{ parameter.python_name }} is not UNSET:
+    cookies["{{ parameter.python_name | kebabcase}}"] = {{ parameter.python_name }}
```

Comment:
```suggestion
    cookies["{{ parameter.name }}"] = {{ parameter.python_name }}
```
Don't you still want parameter.name set here?

Reply:
Sorry, I missed that